solidity/contracts/bridge/Bridge.sol

```diff
@@ -1227,6 +1284,62 @@ contract Bridge is Ownable {
         bank.transferBalance(treasury, outputsInfo.totalTreasuryFee);
     }
 
+    /// @notice Submits a wallet fraud challenge.
```

Comment:
Worth adding more about that function. My proposition:
```
Submits a fraud challenge indicating that a UTXO being under wallet control was unlocked
by the wallet but was not used according to the protocol rules. That means the wallet signed
a transaction input pointing to that UTXO and there is a unique sighash and signature pair 
associated with that input. This function uses those parameters to create a fraud accusation 
that proves a given transaction input unlocking the given UTXO was actually signed by the 
wallet. This function cannot determine whether the transaction was actually broadcast and 
the input was consumed in a fraudulent way so it just opens a challenge period during which 
the wallet can defeat the challenge by submitting proof of a transaction that consumes the given 
input according to protocol rules. To prevent spurious allegations, the caller must deposit 
ETH that is returned back upon justified fraud challenge or confiscated otherwise.
```